src/components/App.tsx

```diff
@@ -111,8 +111,7 @@ export default class App extends PureComponent<Props> {
   render() {
     const { copyAuthenticationHeader, session, logout } = this.props;
     const contentClasses = `col-sm-9 content`;
-    const version =
-      process.env.REACT_APP_VERSION || process.env.KINTO_ADMIN_VERSION;
```

Comment:
This would deserve an entry in the CHANGELOG, as a breaking change?

Reply:
This would be something to include in the release notes of the next release, right? Since we don't have a CHANGELOG in this repo (yet).

Reply:
Ah! I thought you added it ;)